src/test/java/de/retest/web/RecheckSeleniumAdapterTest.java

```diff
@@ -58,6 +58,19 @@ void convertToPeers_should_result_in_valid_tree() throws Exception {
 		assertThat( diffs ).isEmpty();
 	}
 
+	@Test
+	void outline_should_be_added_to_ignored_attributes() {
+		System.setProperty( "de.retest.ignoredAttributes", "foo" );
```

Comment:
If you modify properties in a test, always clean up afterwards (which is what the `SystemProperty` rule in our JUnit 4 tests does). Have a look at this test (another branch): https://github.com/retest/recheck-web/blob/feature/fix-test/src/test/java/de/retest/web/AttributesProviderTest.java#L32